README.md

```diff
@@ -340,3 +340,20 @@ Configurator::apply()
     ->withFileReader('.xml', MyCustomXMLFileReader::class)
     ->configFromFile('config.xml'),
     ->to($container);
+```
+
+### Adding A Custom Container Adapter
+
+You can create our own container adapter so that you can configure other
```

Comment:
Should this be 

```diff
-You can create our own container adapter so that you can configure other
+You can create your own container adapter so that you can configure other
```

instead?